src/client/app/actions/conversions.ts

```diff
@@ -150,7 +150,7 @@ export function deleteConversion(conversion: t.ConversionData): Thunk {
 				dispatch(confirmDeletedConversion(conversion));
 				showSuccessNotification(translate('conversion.successfully.delete.conversion'));
 			} catch (err) {
-				showErrorNotification(translate('conversion.failed.to.delete.conversion'));
+				showErrorNotification(translate('conversion.failed.to.delete.conversion') + '"' + err + '"');
```

Comment:
This was not your doing but this message does not show up as expected because src/server/routes/conversions.js near the end of the file for the delete route does not use failure but res.status. This happens in two places. Would you be willing to fix that?

If you do edit that file then you could also fix the addConversion route that has
validationResult.error.toString()
but it should be
validationResult.errors.toString()
as done elsewhere. I missed that when the code was added to OED in the past. It causes an issue so the message is not seen.

Reply:
I fixed to use success/failure and you did the rest so this is fixed.